scripts/notebooks/Batching.ipynb

```diff
@@ -0,0 +1,1275 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Display columns available to batch on\n",
+    "print(\"Columns available to batch on:\")\n",
+    "print(\"------------------------------\")\n",
+    "\n",
+    "include_cols = [col for col in pass_df.columns if col not in ('sample_id', 'chrX_CopyNumber_rounded')]\n",
+    "for col in include_cols:\n",
+    "    if (not pass_df[col].isnull().all()):\n",
+    "        print(f\"{col}:\")\n",
+    "        print(f\" - Mean: {round(pass_df[col].mean(), 3)}\")\n",
+    "        print(f\" - Median: {round(pass_df[col].median(), 3)}\")\n",
+    "        print(f\" - MAD: {round(np.median(np.abs(pass_df[col] - pass_df[col].median())), 3)}\")\n",
+    "        print()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">Based on the columns listed above, input the list of metrics you would like to batch based on, as well as the weights to give these metrics. We recommend using <tt>median_coverage</tt> and <tt>wgd_score</tt> for batching at minimum. <br><br>You may also wish to batch samples based on other characteristics that could impact SV calling for your data, such as mean insert size or PCR status. If these additional metrics do not appear above, you may add them to the metadata file and reload it.</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "INCLUDE_METRICS = ['TODO']\n",
+    "INCLUDE_METRICS_WEIGHTS = ['TODO']\n",
+    "\n",
+    "validate_include_metrics(INCLUDE_METRICS, INCLUDE_METRICS_WEIGHTS, include_cols)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">Input the list of bins you'd like to divide samples into, where INCLUDE_BINS[i] corresponds to the number of bins for INCLUDE_METRICS[i]. Note that the number of bins for the final metric in INCLUDE_METRICS will be determined based on the other parameters, so INCLUDE_BINS should contain 1 less element than INCLUDE_METRICS.</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "INCLUDE_BINS = ['TODO']\n",
+    "\n",
+    "validate_include_bins(INCLUDE_METRICS, INCLUDE_BINS)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">Input the target batch size you'd like to create, as well as the minimum and maximum batch sizes.</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "TARGET_BATCH_SIZE = 'TODO'\n",
+    "MINIMUM_BATCH_SIZE = 'TODO'\n",
+    "MAXIMUM_BATCH_SIZE = 'TODO'\n",
+    "\n",
+    "validate_numeric_inputs([TARGET_BATCH_SIZE, MINIMUM_BATCH_SIZE, MAXIMUM_BATCH_SIZE])"
```

Comment:
Probably should validate that min size is at most the number of total samples

Reply:
Updated accordingly.